redis.conf

```diff
@@ -286,6 +286,11 @@ pidfile /var/run/redis_6379.pid
 # warning (only very important / critical messages are logged)
 loglevel notice
 
+# To enable raw logging, set this to 'yes'. By default the logging on
+# the 'logfile' contains timestamp, pid and other information, which this
+# option can disable. Helpful when logging to stdout/stderr on systemd.
+lograw no
+
```

Comment:
This make sense. Wouldn't it be better to have a custom log format string where you can put the PID,time format, etc. Or omit them as needed. This is a more generic approach to `lograw`. How complex would it be to take that path?